contracts/core/OTCWrapper.sol

```diff
@@ -520,10 +535,20 @@ contract OTCWrapper is Initializable, OwnableUpgradeable, ReentrancyGuardUpgrade
         require(block.timestamp <= (order.openedAt + fillDeadline), "OTCWrapper: deadline has passed");
         require(whitelist.isWhitelistedCollateral(_collateralAsset), "OTCWrapper: collateral is not whitelisted");
 
+        // notional is expected to have 6 decimals
+        // size and oracle price are expected to have 8 decimals
+        // in aggregate we get 1e16 and therefore divide by 1e10 to obtain 6 decimals
+        uint256 notional = (order.size * oracle.getPrice(order.underlying)) / (1e10);
+
+        require(
```

Comment:
lets remove this min/max notional check

Reply:
As discussed the second min/max notional check was removed and replaced with max deviation to prevent price fluctuations between place order time and execute time to be higher than what we allow
https://github.com/ribbon-finance/GammaProtocol-OTC/pull/5/commits/8c84febfd98daab67732a95832250a7fa0ba5251